### ISSUE TITLE
Failing R CMD check

### ISSUE THREAD

**Issue by jwokaty** [lwaldron/doppelgangR#21]
One of the tests appear to be failing. See https://bioconductor.org/checkResults/3.17/bioc-LATEST/doppelgangR/nebbiolo1-checksrc.html

```
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
       ▆
    1. ├─base::sort(df7a[i, 1:2]) at test_main.R:264:2
    2. └─base::sort.default(df7a[i, 1:2])
    3.   ├─x[order(x, na.last = na.last, decreasing = decreasing)]
    4.   ├─base::`[.data.frame`(x, order(x, na.last = na.last, decreasing = decreasing))
    5.   └─base::order(x, na.last = na.last, decreasing = decreasing)
    6.     └─base::lapply(z, function(x) if (is.object(x)) as.vector(xtfrm(x)) else x)
    7.       └─base (local) FUN(X[[i]], ...)
    8.         ├─base::as.vector(xtfrm(x))
    9.         ├─base::xtfrm(x)
   10.         └─base::xtfrm.data.frame(x)
  
  [ FAIL 1 | WARN 0 | SKIP 0 | PASS 85 ]
  Error: Test failures
```

**Comment by lwaldron** [comment]
I fixed this particular problem in 157493aa9152a5f8d9ee55b92f5b52faed226b7d, but check is [now failing](https://github.com/waldronlab/doppelgangR/actions/runs/5278918809/jobs/9548896069) because of an [error in the curatedOvarianData package](https://github.com/waldronlab/curatedOvarianData/issues/2) that is a Suggests and used in the vignette. Wait to close until doppelgangR is actually building & checking.

**Comment by lwaldron** [comment]
Fixed in RELEASE_3_17 (7d3c7a5) and devel (157493a). Note, curatedOvarianData is OK on bioconductor.org release and devel.
